opds_import.py

```diff
@@ -671,15 +683,32 @@ def summary_to_linkdata(detail):
         return kwargs_meta
 
     @classmethod
-    def rights_uri_for_entry(cls, entry):
+    def rights_uri(cls, rights_string):
         """Determine the URI that best encapsulates the rights status of
         the downloads associated with this book.
+        """
+        return RightsStatus.rights_uri_from_string(rights_string)
 
-        :return: A URI
+    @classmethod
```

Comment:
This is an annoying mess -- both the feedparser entry-creation code and the elementtree entry-creation code need to retrieve the rights URI for an entry, if there is one. In the Feedbooks case, the feedparser code can't determine the rights URI (it depends on information unavailable to feedparser) so the elementtree code has to come in later and fix it.